.github/workflows/tiobe.yaml

```diff
@@ -0,0 +1,39 @@
+name: TICS scan
+
+on:
+  schedule:
+    - cron: "0 12 * * 3" # wednesday at 12:00
+
+jobs:
+  tiobe:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v5
+
+      - name: Create and activate virtual environment
+        run: |
+          python3 -m venv tiobe_venv
+          source tiobe_venv/bin/activate
+          pip install flake8 pylint pytest tox
+          echo PATH="$PATH" >> "$GITHUB_ENV"
+
+      - name: Run coverage tests
+        run: |
+          tox -e unit
+
+      - name: Move results to the necessary folder for TICS
+        run: |
+          mkdir -p .cover
+          mv coverage.xml .cover/cobertura.xml
+
+      - name: TICS GitHub Action
+        uses: tiobe/tics-github-action@v3
+        with:
+          mode: qserver
+          project: ubuntu-pro-operator
+          viewerUrl: https://canonical.tiobe.com/tiobeweb/TICS/api/cfg?name=default
+          branchdir: ${{ env.GITHUB_WORKSPACE }}
+          ticsAuthToken: ${{ secrets.TICSAUTHTOKEN }}
+          installTics: true
+          calc: ALL
```

Comment:
Let's make this match exactly the kind of pattern we have in our other projects.